PSReadLine/YankPaste.vi.cs

```diff
@@ -70,6 +70,26 @@ private void SaveLinesToClipboard(int lineIndex, int lineCount)
             _clipboard.LinewiseRecord(_buffer.ToString(range.Offset, range.Count));
         }
 
+        /// <summary>
+        /// Removes a portion of text from the buffer
+        /// and saves it to the clipboard in order to support undo.

```

Comment:
I think the undo is supported by `_singleton.SaveEditItem`, not `_singleton.SaveToClipboard`. Maybe update the comment to:
```suggestion
        /// Remove a portion of text from the buffer,  save it to the vi register,
        /// and also save it to the edit list to support undo.
```